content/blog/2024-06-18-unikraft-gsoc-lxboot_x86.mdx

```diff
@@ -0,0 +1,112 @@
+| 0211/1      | 2.00+       | loadflags           | Boot protocol option flags                             |
+| 0212/2      | 2.00+       | setup_move_size     | Move to high memory size (used with hooks)             |
+| 0214/4      | 2.00+       | code32_start        | Boot loader hook (see below)                           |
+| 0218/4      | 2.00+       | ramdisk_image       | initrd load address (set by boot loader)               |
+| 021C/4      | 2.00+       | ramdisk_size        | initrd size (set by boot loader)                       |
+| 0220/4      | 2.00+       | bootsect_kludge     | DO NOT USE - for bootsect.S use only                   |
+| 0224/2      | 2.01+       | heap_end_ptr        | Free memory after setup end                            |
+| 0226/1      | 2.02+(3)    | ext_loader_ver      | Extended boot loader version                           |
+| 0227/1      | 2.02+(3)    | ext_loader_type     | Extended boot loader ID                                |
+| 0228/4      | 2.02+       | cmd_line_ptr        | 32-bit pointer to the kernel command line              |
+| 022C/4      | 2.03+       | initrd_addr_max     | Highest legal initrd address                           |
+| 0230/4      | 2.05+       | kernel_alignment    | Physical addr alignment required for kernel            |
+| 0234/1      | 2.05+       | relocatable_kernel  | Whether kernel is relocatable or not                   |
+| 0235/1      | 2.10+       | min_alignment       | Minimum alignment, as a power of two                   |
+| 0236/2      | 2.12+       | xloadflags          | Boot protocol option flags                             |
+| 0238/4      | 2.06+       | cmdline_size        | Maximum size of the kernel command line                |
+| 023C/4      | 2.07+       | hardware_subarch    | Hardware subarchitecture                               |
+| 0240/8      | 2.07+       | hardware_subarch_data| Subarchitecture-specific data                          |
+| 0248/4      | 2.08+       | payload_offset      | Offset of kernel payload                               |
+| 024C/4      | 2.08+       | payload_length      | Length of kernel payload                               |
+| 0250/8      | 2.09+       | setup_data          | 64-bit physical pointer to linked list of struct setup_data|
+| 0258/8      | 2.10+       | pref_address        | Preferred loading address                              |
+| 0260/4      | 2.10+       | init_size           | Linear memory required during initialization           |
+| 0264/4      | 2.11+       | handover_offset     | Offset of handover entry point                         |
+
+For more details about certain fields, read [here](https://www.kernel.org/doc/html/v5.4/x86/boot.html).
+
+### Next Steps
+
+Once everything works for QEMU, I will move onto GRUB2 and look into making lxboot x86 work with it as ISO and as Disk.
+One difference between two different boot loaders is that some have an identifier.
```

Comment:
This sentence is a bit confusing. Would something like "Boot loaders have unique reserved identifiers, placed in the `type_of_loader` field" make more sense?